### PR TITLE
Index features section view

### PR DESCRIPTION
**Pull request by ziflit**
## Summary
Adds the 'features' middle section for the view.
## Trello card
https://trello.com/c/iCt4qFOc
### Desktop
index-features-section-view 1
### Mobile
index-features-section-view 2

## Commentary
This branch includes changesets from #5

### CODE REVIEW

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
-  padding-top: 40px;
+}
+
+.main-section {
+  @extend %flex-center;
+  border-top: 5px solid $light-orange;
+  padding: 40px 0;
+}
+
+.features-section::before {
```
Comment (SKOLZ) [inline app/assets/stylesheets/base.scss]: this should be nested inside the features-section selector like this:
```
.features-section {
  ...

  &::before {
    ...
  }
}

File: app/assets/stylesheets/base.scss [hunk 2]
```diff
@@ -1,14 +1,55 @@
...
+  top: -12px;
+  left: 0;
+  width: 100%;
+  height: 12px;
+  background: asset-url('triangle.png');
+}
+
+.features-section {
+  position: relative;
+  @extend %flex-center;
```
Comment (SKOLZ) [inline app/assets/stylesheets/base.scss]: `@extend` should be the first selector property

File: app/assets/stylesheets/base.scss [hunk 3]
```diff
@@ -1,14 +1,55 @@
...
+  padding: 50px 0 25px 0;
```
Comment (SKOLZ) [inline app/assets/stylesheets/base.scss]: `padding: 50px 0 25px` should work fine

File: app/assets/stylesheets/base.scss [hunk 4]
```diff
@@ -1,14 +1,55 @@
...
+  justify-content: space-around;
+  background: $light-gray;
+  align-items: baseline;
+}
+
+.feature {
+  width: 20%;
```
Comment (SKOLZ) [inline app/assets/stylesheets/base.scss]: we can use:
```
width: 100%;
max-width: 300px;
```

that way the responsiveness would be easier. You can change the 300px to a pixel value that fits better.

File: app/assets/stylesheets/variables/colors.scss [hunk 5]
```diff
@@ -1,3 +1,5 @@
-$white: white;
+$white: #ffffff;
```
Comment (SKOLZ) [inline app/assets/stylesheets/variables/colors.scss]: colors that look like #AABBCC should be written like this: #ABC. In this case `#FFF` would work.

File: app/assets/stylesheets/variables/colors.scss [hunk 6]
```diff
@@ -1,3 +1,5 @@
...
 $orange: #f4a641;
 $light-orange: #f4b841;
+$light-gray: #EEEEEE;
```
Comment (SKOLZ) [inline app/assets/stylesheets/variables/colors.scss]: #EEE

File: app/assets/stylesheets/variables/colors.scss [hunk 7]
```diff
@@ -1,3 +1,5 @@
...
+$medium-gray: #777777;
```
Comment (SKOLZ) [inline app/assets/stylesheets/variables/colors.scss]: #777

File: app/views/landing/index.html.slim [hunk 8]
```diff
@@ -3,7 +3,26 @@ html
...
   body
-    .container
+    article.main-section
       = image_tag('phone.png', class: 'phone')
       .vertical-separator
       = image_tag('tcg_logo.png', class: 'logo')
+    article.features-section
+      .feature
+        = image_tag('ic_list.png', class: 'feature-image')
+        b
```
Comment (SKOLZ) [inline app/views/landing/index.html.slim]: don't use `b` tags, b stans for bold and style stuff should be handled through css. use a headline tag instead

File: app/assets/stylesheets/base.scss [hunk 9]
```diff
@@ -1,14 +1,58 @@
...
+
+.feature {
+  width: 100%;
+  max-width: 220px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  text-align: center;
+  h4 {
```
Comment (ziflit) [inline app/assets/stylesheets/base.scss]: not quite sure if this belongs here.

File: app/assets/stylesheets/base.scss [hunk 10]
```diff
@@ -1,14 +1,58 @@
...
+}
+
+.features-section {
+  @extend %flex-center;
+  position: relative;
+  padding: 50px 0 25px;
+  justify-content: space-around;
+  background: $light-gray;
+  align-items: baseline;
+  &::before {
```
Comment (SKOLZ) [inline app/assets/stylesheets/base.scss]: leave an empty line between this selector and the properties above

File: app/assets/stylesheets/base.scss (hunk 9 above)
Comment (SKOLZ) [inline app/assets/stylesheets/base.scss]: don't target html elements, use classes instead

**Pull request merged**